core/ts/MaciState.ts

```diff
@@ -601,8 +607,6 @@ class Poll {
             deactivatedKeyEvent.c1,
             deactivatedKeyEvent.c2
         )
```

Comment:
Why we don't return the `circuitInputs` along with the `kcommand` anymore?

Reply:
We are returning const { circuitInputs, encPubKey, message }  from the call directly to the caller.